<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gondericiEkle</class>
 <widget class="QWidget" name="gondericiEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gönderici Ekle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
 	 background: #3498db;
	border-radius: 18px;
	font-family: Courier New;
	color: #ffffff;
	font-size: 16px;
	padding: 10px 20px 10px 20px;
	text-decoration: none;
    max-width:120px;
	min-width:120px;
	max-height:20px;	 
	min-height:20px;	 
}

QPushButton:hover {
	background: #3cb0fd;
	text-decoration: none;
}
QLineEdit{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QComboBox{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QWidget{
		font-family: Courier New;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>641</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>55</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblAd">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px </string>
        </property>
        <property name="text">
         <string>Ad :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblSoyad">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px</string>
        </property>
        <property name="text">
         <string>Soyad :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblCepNo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px 
</string>
        </property>
        <property name="text">
         <string>Cep No :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblAdres">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px </string>
        </property>
        <property name="text">
         <string>Adres :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblSube">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px </string>
        </property>
        <property name="text">
         <string>Şube :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblMail">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:bold 18px</string>
        </property>
        <property name="text">
         <string>Mail :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lnAd">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
font:16px
</string>
        </property>
        <property name="placeholderText">
         <string>Adınızı giriniz..</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lnSoyad">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
font:16px 
</string>
        </property>
        <property name="placeholderText">
         <string>Soyadınızı giriniz..</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lnCepNo">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
font:16px </string>
        </property>
        <property name="inputMask">
         <string>(999) 999 99 99</string>
        </property>
        <property name="text">
         <string>()   </string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lnAdres">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
border-radius:10px;
font:16px </string>
        </property>
        <property name="placeholderText">
         <string>Adresinizi giriniz..</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="cmbxSubeList">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
font:16px 


</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lnMail">
        <property name="minimumSize">
         <size>
          <width>440</width>
          <height>31</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:10px;
font:16px </string>
        </property>
        <property name="placeholderText">
         <string>Mailinizi lütfen doğru giriniz..</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_geri">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Geri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKaydet">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
  <zorder>btn_geri</zorder>
  <zorder>btnKaydet</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
